testing/vault/vault.pre-deinstall

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# Stop vault service
+rc-service vault stop &> /dev/null
```

Comment:
`&>` is not POSIX-sh compatible, it’s a _bashism_. I also don’t think that it’s needed, user should now that (s)he should stop the service before uninstalling…